bundles/org.openhab.binding.hue/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -133,6 +141,10 @@
 		<item-type>String</item-type>
 		<label>Alert</label>
 		<description>The alert channel allows a temporary change to the bulb’s state.</description>
+		<tags>
+			<tag>Control</tag>
+			<tag>Alarm</tag>
```

Comment:
Is this combination valid?

Reply:
You're right, I was just checking the semantics, they are both "Point".
I am going to suppress "Control".

Reply:
Fixed